### ISSUE TITLE
Please build intermediate older releases

### ISSUE THREAD

**Issue by tpdownes** [conda-forge/terraform-feedstock#84]
- [X] I read [the conda-forge documentation](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge) and could not find the solution for my problem there.

**Issue**: the [Terraform registry recently updated its GPG signing key][discuss] necessitating backports to prior releases of Terraform which are still in community use. This key allows Terraform to download recent updates to providers like Google, AWS, etc.

[discuss]: https://discuss.hashicorp.com/t/terraform-updates-for-hcsec-2021-12/23570 

Can you please trigger builds for these versions?

* [v0.14.11](https://github.com/hashicorp/terraform/blob/v0.14/CHANGELOG.md#01411-april-26-2021)
* [v0.13.7](https://github.com/hashicorp/terraform/blob/v0.13/CHANGELOG.md#0137-april-26-2021)
* [v0.12.31](https://github.com/hashicorp/terraform/blob/v0.12/CHANGELOG.md#01231-april-26-2021)
* [v0.11.15](https://github.com/hashicorp/terraform/blob/v0.11/CHANGELOG.md#01115-april-26-2021)

**Comment by tpdownes** [comment]
There are a few other missing backports (e.g. 0.12.30 and 0.13.6). These do not matter so much as their fixes will be included in the ones I'm requesting. But you might consider them for completeness.

**Comment by tpdownes** [comment]
Any thoughts here?

@dhirschfeld
@mariusvniekerk
@ndmaxar
@oblute
@rluria14
@sodre

**Comment by tpdownes** [comment]
Here is the error you get using 0.12.29. The hidden consequence is that you are stuck on a version of the Google TF provider from April despite newer releases being perfectly compatible.

```
Warning: registry.terraform.io: This version of Terraform has an outdated GPG key and is unable to verify new provider releases. Please upgrade Terraform to at least 0.12.31 to receive new provider updates. For details see: https://discuss.hashicorp.com/t/hcsec-2021-12-codecov-security-event-and-hashicorp-gpg-key-exposure/23512
```

**Comment by mariusvniekerk** [comment]
I have created branches for the last builds for 0.11--0.15.  If you would like build for these old release please submit a pull-request targeting those branches

**Comment by tpdownes** [comment]
@mariusvniekerk Let me know if the PR for 0.12 needs work. Thanks! I'll follow up with others if it looks alright.

**Comment by tpdownes** [comment]
Resolved by #91 #93 #94 #95
